amplipi/updater/static/index.html

```diff
@@ -105,6 +110,36 @@ <h1><span class="text-white">Ampli</span><span class="text-danger">Pi</span></h1
                 </div>
               </div>
             </div>
+
+            <div class="tab-pane fade text-left" id="admin-settings" role="tabpanel" aria-labelledby="admin-settings-tab">
+              <script>
+                async function getPersist() {
+                  const response = await fetch("/settings/persist_logs", { method: "GET" });
+                  const data = await response.json();
+                  let checkbox = document.getElementById("persist_logs_checkbox");
+                  checkbox.checked = data;
+                }
+
+                async function setPersist() {
+                  const checkbox = document.getElementById("persist_logs_checkbox");
+                  const response = await fetch("/settings/persist_logs", {method: "POST"});
+                }
+
+                window.onload = getPersist;
+              </script>
+
+              <div id="admin-settings-dialog">
+                This tab allows you to change settings relating to the core of AmpliPi, things that do not belong on the standard config page
+              </div>
+              <p class="">
+                <a class="btn btn-primary btn-sm text-center" id="set-password-button" data-toggle="tab" href="#set-password" role="tab" aria-controls="set-password" aria-selected="false">Set Password</a>
+              </p>
+              <h6>WARNING: leaving log persistence on for extended periods can damage your system and void your warranty</h6>
```

Comment:
is this true?

Reply:
I've been lead to believe that persisting logs will lead to an incredible amount of reads and writes and wear out the drive, if a customer chooses to knowingly do that to themselves I can't imagine we'd cover that; I haven't passed this line by anyone yet however

That said, I don't actually track how long persist_logs has been active and so a worn storage device doesn't necessarily point towards any culprit

Reply:
warranties are legally binding, and thus we should be explicit about what exactly voids the warranty. if turning this setting on voids it, great, but specifying an indeterminate amount of time is probably not ideal.

ime, running a raspberry pi for years with vanilla journalctl hasn't worn out my eMMC. It's a good idea to limit writes, but I hesitate to say we should void warranties about this. the default case is good for limiting writes, and if people want to turn it on they should be able to.

@linknum23 do you have thoughts on this one particular line?

Reply:
> ime, running a raspberry pi for years with vanilla journalctl hasn't worn out my eMMC

This gives me the sense that the line is unnecessary as the warranty will be expired by the time it matters. I can probably forgo the warning at that rate as anyone touching this setting ought to understand what that entails and want logs to stick around; I can't imagine a user that doesn't know what logs are would go persisting them.

I'll await Lincoln's two cents before removing anything, though

Reply:
We need to find a simpler way to do this without the scary warning. Maybe only enabling logging for 2 weeks?

Reply:
Instead of changing the setting automatically, would it suffice to have the frontend occasionally pop a notification that's like "hey, look out, this setting is still on!" And have it be colored yellow instead of the green/red the other notifications are

Perhaps not due to the number of people that only use a custom API caller to interact with the device, but I feel Ryan's instinct (or we least what I've interpreted his instinct to be) that we should leave this high level control in the hands of the device owners rather than trying to know better than them is for the best. Inform where necessary, but leave them to do their thing without any bothersome automations getting in the way of them doing what they want to do with their own system(s).